panther_bringup/launch/bringup.launch.py

```diff
@@ -239,52 +128,39 @@ def generate_launch_description():
         package="panther_diagnostics",
         executable="system_status",
         name="system_status",
-        output="screen",
-        condition=UnlessCondition(use_sim),
         namespace=namespace,
         remappings=[("/diagnostics", "diagnostics")],
+        condition=UnlessCondition(use_sim),
     )
 
     lights_launch = IncludeLaunchDescription(
         PythonLaunchDescriptionSource(
             PathJoinSubstitution(
-                [
-                    FindPackageShare("panther_lights"),
-                    "launch",
-                    "lights.launch.py",
-                ]
+                [FindPackageShare("panther_lights"), "launch", "lights.launch.py"]
             )
         ),
         condition=UnlessCondition(use_sim),
         launch_arguments={
-            "led_config_file": led_config_file,
             "namespace": namespace,
-            "user_led_animations_file": user_led_animations_file,
         }.items(),
     )
 
     battery_launch = IncludeLaunchDescription(
         PythonLaunchDescriptionSource(
             PathJoinSubstitution(
-                [
-                    FindPackageShare("panther_battery"),
-                    "launch",
-                    "battery.launch.py",
-                ]
+                [FindPackageShare("panther_battery"), "launch", "battery.launch.py"]
             ),
         ),
         condition=UnlessCondition(use_sim),
         launch_arguments={
             "namespace": namespace,
-            "panther_version": panther_version,
         }.items(),
     )
 
     robot_localization_node = Node(
         package="robot_localization",
         executable="ekf_node",
         name="ekf_node",
-        output="screen",
         parameters=[ekf_config_path, {"tf_prefix": namespace}],
         namespace=namespace,
         remappings=[
```

Comment:
How about creating a separate package `panther_robot_localization` for ekf things? We would store there config and launch file so from here ekf_launch will be included. Bring up as long as possible shouldn't define Nodes itself (I know there is still system_status node but it'll be changed in one of Dawid's upcoming PR).

Reply:
IMO good idea. Also we get cleaner bringup, thanks to move declaration args.

Reply:
I meant a truly separate package - sth as we got for ros2_control  -> `panther_controller`.

Reply:
Maybe `panther_ekf`?

Reply:
`panther_localization` sounds the best 🏆

The name doesn't narrow the specific localization technique so we're able to add e.g. second launch and configuration for GPS setup in the future.

Reply:
With gps it's `robot_localization` pkg still use Kalman Filter, so is the most correct one name should be related with **kalman filter**, as we use extended kalman filter and there is no plan to change it right now `ekf` is the most concrete name for this pkg. In future of course we can change the name if we decide to use other filtration metod.

`localization` in my opinion, is too broad a word and may as well catching the AMCL algorithm there and I not sure if this is what we want.